cesium/features/lomb_scargle.py

```diff
@@ -25,6 +26,13 @@ def lomb_scargle_model(time, signal, error, sys_err=0.05, nharm=8, nfreq=3, tone
     nfreq : int
         Number of frequencies to fit.
 
+    freq_grid : str
```

Comment:
Looks like this can be one argument: freq_grid. If specified, it is used, otherwise determined automatically.

Reply:
done